setup.cfg

```diff
@@ -88,9 +88,25 @@ addopts =
 # Why do we exclude this file from coverage?
 # Because coverage is not calculated correctly for pytest plugins.
 # And we completely test it anyway.
+branch = True
+data_file = .tests_reports/.coverage
 omit =
   django_test_migrations/contrib/pytest_plugin.py
 
+[coverage:report]
+skip_covered = True
```

Comment:
The same is configured here: https://github.com/wemake-services/django-test-migrations/blob/686d03a6e4898db8b6e49177b82deda916976f45/setup.cfg#L81